### ISSUE TITLE
Fedora gating: inconsistent quotes around invalid duration

### ISSUE THREAD

**Issue by edsantiago** [containers/buildah#2773]
Bodhi gating tests are [failing](https://osci-jenkins-1.ci.fedoraproject.org/job/fedora-ci/job/dist-git-pipeline/job/master/3547/testReport/(root)/tests/root_test/):
```
not ok 237 config-flags-verification
# (from function `expect_output' in file /usr/share/buildah/test/system/helpers.bash, line 265,
#  in test file /usr/share/buildah/test/system/config.bats, line 32)
#   `expect_output --substring 'error parsing --healthcheck-interval "ABCD": time: invalid duration ABCD'' failed with status 125
# /usr/share/buildah/test/system /var/str
# $ /usr/bin/buildah from --signature-policy /usr/share/buildah/test/system/policy.json scratch
# working-container
# $ /usr/bin/buildah config --label LABEL working-container
# error adding label "LABEL": no value given
# [ rc=125 (expected) ]
# $ /usr/bin/buildah config --annotation ANNOTATION working-container
# error adding annotation "ANNOTATION": no value given
# [ rc=125 (expected) ]
# $ /usr/bin/buildah config --healthcheck AB "CD working-container
# error parsing --healthcheck "AB \"CD": invalid command line string
# [ rc=125 (expected) ]
# $ /usr/bin/buildah config --healthcheck-interval ABCD working-container
# error parsing --healthcheck-interval "ABCD": time: invalid duration "ABCD"
# [ rc=125 (expected) ]
# #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
# #|     FAIL: buildah config --healthcheck-interval ABCD working-container
# #| expected: 'error parsing --healthcheck-interval "ABCD": time: invalid duration ABCD'
# #|   actual: 'error parsing --healthcheck-interval "ABCD": time: invalid duration "ABCD"'
# #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# /var/str
```
Culprit: new test added in #2753 expects the invalid duration to be as-is; actual error message quotes it.

As best I can tell, this seems to be a Go version difference: `time.ParseDuration()` appears to have changed the format of its output messages. 

Recommendation: the `expect_output` may need to be tweaked a little to handle optional quotes. Not sure, I'm still trying to confirm my guess.

**Comment by rhatdan** [comment]
Can you just change it to look for "time: invalid duration"?

**Comment by rhatdan** [comment]
@edsantiago Does this issue still exist?

**Comment by edsantiago** [comment]
Yes, still failing in yesterday's [buildah-1.19.4-1.fc33](https://osci-jenkins-1.ci.fedoraproject.org/job/fedora-ci/job/dist-git-pipeline/job/master/18169/testReport/(root)/tests/root_test/)

**Comment by edsantiago** [comment]
Fixed by #2631
